assets/js/blocks/collection-filters/inner-blocks/price-filter/block.json

```diff
@@ -10,19 +10,22 @@
 	],
 	"textdomain": "woo-gutenberg-products-block",
 	"apiVersion": 2,
-	"viewScript": [
-		"wc-collection-price-filter-block-frontend"
-	],
 	"ancestor": [
-		"woocommerce/product-collection"
+		"woocommerce/collection-filters"
```

Comment:
Should this still have the word product in it? i have to admit the vagueness caught me a few times. I think for the way we're organizing the code this would always be just for product data? Not fussed since its still experimental but thought I'd mention it.

Reply:
> Should this still have the word product in it?

Do you mean we should rename the `woocommerce/colleciton-filters` to `woocommerce/product-collection-filters`?

Reply:
Yes thats what I meant, is that correct that this will only be for products?

Reply:
Makes sense to me. I created https://github.com/woocommerce/woocommerce-blocks/issues/11643 to work on that further as this PR is already quite big.